themes/default/content/docs/intro/pulumi-service/billing-managers.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Billing Managers"
+meta_desc: Overview of Billing Managers in the Pulumi Service.
+menu:
+  intro:
+    parent: pulumi-service
+    weight: 11
+---
+
+The Billing Manager role provides customers the ability to have someone in your organization manage billing operations without granting them any additional permissions to view or modify your stacks, policies, or other organizational settings.
+
+Any organization admin can view, invite, and remove Billing Managers from their organization.
+
+Team Editions of the Pulumi Service are limited to 1 Billing Manager at a time. Enterprise and Business Critical subscriptions can have unlimited Billing Managers in their organization.
+
+## Inviting a Billing Manager to an Organization
+
+Billing Manager invitations work much the same way member invitations do, but they are managed on the Billing and Usage page.
+
+To invite a new Billing Manager:
+
+1. Select **Settings**.
+1. Select **Billing & usage**.
+1. To invite a new Billing Manager using their email address, select **Invite Billing Managers**. After the new Billing Manager is invited, they will receive an email allowing them to accept the invite.
+1. To invite a new Billing Manager by sharing a link, select **Copy Billing Manager invite link**. This will copy a link to your clipboard that you can share directly.
```

Comment:
```suggestion
4. To invite a new Billing Manager by sharing a link, select **Copy Billing Manager invite link**. This will copy a link to your clipboard that you can share directly.
```